src/components/widget-list-card.jsx

```diff
@@ -0,0 +1,259 @@
+
+    let widgetErrorsRender = null
+    if (state.errorMessage) {
+        widgetErrorsRender = state.errorMessage.map(error => <div className="error-holder">{error}</div>)
+    }
+
+    let widgetSuccessRender = null
+    if (state.success) {
+        widgetSuccessRender = <div className="success-holder">Widget Saved!</div>
+    }
+
+    let featuresRender = null
+    if (state.widget.meta_data.features) {
+        featuresRender = state.widget.meta_data.features.map((feature, i) => <li>{ feature }</li>)
+    }
+
+    let questionTypes = null
+    if (state.widget.meta_data.supported_data) {
+        questionTypes = state.widget.meta_data.supported_data.map((qtype, i) => <li>{ qtype }</li>)
+    }
+
+    let exportOptions = null
+    if (state.widget.meta_data.playdata_exporters) {
+        exportOptions = state.widget.meta_data.playdata_exporters.map((qtype, i) => <li>{ qtype }</li>)
+    }
+
+    return (
+        <li>
+            <div className="clickable widget-title" onClick={handleWidgetClick}>
+                <span className="img-holder">
+                    <img src={state.widget.icon}/>
+                </span>
+                <span className="title">{state.widget.name}</span>
+            </div>
+            { ! state.widget.expanded ? <></> : 
+            <div className="widget-info">
+                { widgetErrorsRender }
+                { widgetSuccessRender }
+                <div className='info-holder'>
+                    <div>
+                        <span>
+                            <label>ID:</label>{ state.widget.id }
+                        </span>
+                    </div>
+                    <div>
+                        <span>
+                            <label>Installed:</label>{ state.widget.created_at /* * 1000 | date:yyyy-MM-dd */ }
+                        </span>
+                    </div>
+                    <div>
+                        <span>
+                            <label>Dimensions:</label>{ state.widget.width }w x { state.widget.height }h
+                        </span>
+                    </div>
+                    <div>
+                        <span>
+                            <label>Settings:</label>
+                        </span>
+                        <span>
+                            <div>
+                                <label className="normal">
+                                    <input type="checkbox" defaultChecked={state.widget.in_catalog} checked={state.widget.in_catalog}
+                                    onClick={toggleInCatalog}/>
+                                    In Catalog
+                                </label>
+                            </div>
+                            <div>
+                                <label className="normal">
+                                    <input type="checkbox" defaultChecked={state.widget.is_editable} checked={state.widget.is_editable}
+                                    onClick={toggleIsEditable}/>
```

Comment:
In addition to the feedback above, I think these inputs should be using `onChange` instead of `onClick` - even if it appears to work just fine as it is now.